index.html

```diff
@@ -15,219 +15,223 @@
-      </li>
-      <li>
-        <a href="samples/audio/oscillator.html">oscillator.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/panning.html">panning.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/pingpong-delay.html">pingpong-delay.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/playthrough.html">playthrough.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/samplerate.html">samplerate.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/samples.html">samples.html</a>
-      </li>
-      <li>
-        <a href=
-        "samples/audio/shiny-drum-machine.html">shiny-drum-machine.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/simple-mixing.html">simple-mixing.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/simple.html">simple.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/stereopanner.html">stereopanner.html/</a>
-      </li>
-      <li>
-        <a href="samples/audio/subtractive.html">subtractive.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/synthetic-clock.html">synthetic-clock.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/test-audio-tag.html">test-audio-tag.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/test-tones.html">test-tones.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/test-waveshaper.html">test-waveshaper.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/timeline.html">timeline.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/tone-editor.html">tone-editor.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/tone.html">tone.html</a>
-      </li>
-      <li>
-        <a href=
-        "samples/audio/visualizer-chorus.html">visualizer-chorus.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/visualizer-gl/">visualizer-gl/</a>
-      </li>
-      <li>
-        <a href="samples/audio/visualizer-gl.html">visualizer-gl.html</a>
-      </li>
-      <li>
-        <a href=
-        "samples/audio/visualizer-live-doubling.html">visualizer-live-doubling.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/visualizer-live.html">visualizer-live.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/visualizer-noise.html">visualizer-noise.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/visualizer2-gl.html">visualizer2-gl.html</a>
-      </li>
-      <li>
-        <a href=
-        "samples/audio/waveshaper-oversample.html">waveshaper-oversample.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/wavetable-synth.html">wavetable-synth.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/wavetable-synth2.html">wavetable-synth2.html</a>
-      </li>
-      <li>
-        <a href="samples/audio/zero-crossings.html">zero-crossings.html</a>
-      </li>
-    </ul>
+    <section>
```

Comment:
What's happening here? Hard to tell from the diff.

Reply:
I've added `<section>` elements. So the indentation is changed.